study-builder/studies/pancan/prequal.conf

```diff
@@ -143,83 +143,15 @@
-            "templateText": """<p class="ddp-question-prompt">$prompt *</p>"""
-            "variables": [
-              {
-                "name": "prompt",
-                "translations": [
-                  { "language": "en", "text": ${i18n.en.prequal.advanced_breast_prompt_self} },
-                  { "language": "es", "text": ${i18n.es.prequal.advanced_breast_prompt_self} },
-                ]
-              }
-            ]
-          },
-          "blockType": "CONTENT",
-          "shownExpr": ${_pex.is_self}"&&"${_pex.has_breast}
-        },
-        {
-          "bodyTemplate": {
-            "templateType": "HTML",
-            "templateText": """<p class="ddp-question-prompt">$prompt *</p>"""
-            "variables": [
-              {
-                "name": "prompt",
-                "translations": [
-                  { "language": "en", "text": ${i18n.en.prequal.advanced_breast_prompt_child} },
-                  { "language": "es", "text": ${i18n.es.prequal.advanced_breast_prompt_child} },
-                ]
-              }
-            ]
-          },
-          "blockType": "CONTENT",
-          "shownExpr": ${_pex.is_child_only}"&&"${_pex.has_breast}
-        },
         {
           "question": ${_includes.question_advanced_breast},
           "blockType": "QUESTION",
-          "shownExpr": ${_pex.has_breast}
-        },
-        {
-          "bodyTemplate": {
-            "templateType": "HTML",
-            "templateText": """<p class="ddp-question-prompt">$prompt *</p>"""
-            "variables": [
-              {
-                "name": "prompt",
-                "translations": [
-                  { "language": "en", "text": ${i18n.en.prequal.advanced_prostate_prompt_self} },
-                  { "language": "es", "text": ${i18n.es.prequal.advanced_prostate_prompt_self} },
-                ]
-              }
-            ]
-          },
-          "blockType": "CONTENT",
-          "shownExpr": ${_pex.is_self}"&&"${_pex.has_prostate}
-        },
-        {
-          "bodyTemplate": {
-            "templateType": "HTML",
-            "templateText": """<p class="ddp-question-prompt">$prompt *</p>"""
-            "variables": [
-              {
-                "name": "prompt",
-                "translations": [
-                  { "language": "en", "text": ${i18n.en.prequal.advanced_prostate_prompt_child} },
-                  { "language": "es", "text": ${i18n.es.prequal.advanced_prostate_prompt_child} },
-                ]
-              }
-            ]
-          },
-          "blockType": "CONTENT",
-          "shownExpr": ${_pex.is_child_only}"&&"${_pex.has_prostate}
+          "shownExpr": ${_pex.is_self_only}"&&"${_pex.has_breast}
         },
         {
           "question": ${_includes.question_advanced_prostate},
           "blockType": "QUESTION",
-          "shownExpr": ${_pex.has_prostate}
+          "shownExpr": ${_pex.is_self_only}"&&"${_pex.has_prostate}
```

Comment:
Wait... when they select both "self" and "child" diagnosed options, the behavior is that we treat it as if it's just "self". That's why I had put in checks like `is_self` and `is_child_only`. If they selected both "self" and "child", I think we should still show advanced breast/prostate questions, since it's technically a "self" case?

Reply:
will update to is_self